recap-loops/index.md

```diff
@@ -183,7 +183,32 @@ for (int number : numbers) {
 
 
 ## Exercises
-### Exercise 1: Multiplication Table
+
+### Sum of an array.
+
+Write a method that takes an array of integers and returns the sum of the elements of said array.
```

Comment:
adding instructions on how to test it (+ introducing some engineering vocab) and simplified the language a bit to be more inclusive to english beginners
```suggestion
Write a method that takes an array of integers and returns the sum of the elements of that array. Test it by passing in a few `hardcoded` inputs and printing the output to the console. 
Examples:
`1,2,3,4] `-> 10
`[]` -> 0
`[1]` -> 1
```